package.json

```diff
@@ -29,6 +29,7 @@
   },
   "devDependencies": {
     "eslint": "^8.2.0",
+    "eslint-9": "npm:eslint@^9.0.0",
```

Comment:
because your commit will be a major version, I recommend to upgrade only the `eslint`